cmake/Modules/FindWayland.cmake

```diff
@@ -0,0 +1,66 @@
+# Try to find Wayland on a Unix system
```

Comment:
I'm not sure why you copied over this version of the FindWayland module. If you use the last version you'll get to use Wayland::Egl as a target removing quite some boilerplate code in the actual use of the module.

https://invent.kde.org/frameworks/extra-cmake-modules/-/blob/master/find-modules/FindWayland.cmake